felix/fv/wireguard_test.go

```diff
@@ -1775,6 +1806,9 @@ func wireguardTopologyOptions(routeSource string, ipipEnabled, wireguardIPv4Enab
 	if wireguardIPv6Enabled {
 		felixConfig.Spec.WireguardEnabledV6 = &enabled
 	}
+	if wireguardThreadingEnabled {
+		felixconfig.Spec.WiregauardThreadingEnabled = &enabled
```

Comment:
`felixConfig`. The linter failed. Please make sure `make static-check` under 'felix' directory passes.

Reply:
Thanks, the test appeared to pass with the latest commit fixing the typos.